scripts/install_ddev.sh

```diff
@@ -117,29 +124,38 @@ else
     exit 1
 fi
 
-USE_ARCH=$(semver_compare "${VERSION}" "v1.16.0-alpha4")
+if [ "$VERSION" == 'HEAD' ]; then
+  FILEBASE="${FILEBASE//-/_}"
+  USE_ARCH=1
+else
+  USE_ARCH=$(semver_compare "${VERSION}" "v1.16.0-alpha4")
+fi
 # Versions after v1.16.0-alpha4 need the architecture in the filename
 if [ "${USE_ARCH}" == 1 ]; then
   FILEBASE="${FILEBASE}-${ARCH}"
 fi
 
-
 if ! docker --version >/dev/null 2>&1; then
     printf "${YELLOW}Docker is required for ddev. Please see https://ddev.readthedocs.io/en/stable/#docker-installation.${RESET}\n"
 fi
 
-TARBALL="$FILEBASE.$VERSION.tar.gz"
-SHAFILE="$TARBALL.sha256.txt"
+if [ "$VERSION" == 'HEAD' ]; then 
+  TARBALL="$FILEBASE.zip"
+else
+  TARBALL="$FILEBASE.$VERSION.tar.gz"
+  SHAFILE="$TARBALL.sha256.txt"
+fi
 
 curl -fsSL "$RELEASE_BASE_URL/$TARBALL" -o "${TMPDIR}/${TARBALL}" || (printf "${RED}Failed downloading $RELEASE_BASE_URL/$TARBALL${RESET}\n" && exit 108)
-curl -fsSL "$RELEASE_BASE_URL/$SHAFILE" -o "${TMPDIR}/${SHAFILE}" || (printf "${RED}Failed downloading $RELEASE_BASE_URL/$SHAFILE${RESET}\n" && exit 109)
+if [ "$VERSION" != 'HEAD' ]; then
+  curl -fsSL "$RELEASE_BASE_URL/$SHAFILE" -o "${TMPDIR}/${SHAFILE}" || (printf "${RED}Failed downloading $RELEASE_BASE_URL/$SHAFILE${RESET}\n" && exit 109)
+fi
 curl -fsSL "https://raw.githubusercontent.com/${GITHUB_USERNAME}/ddev/master/scripts/macos_ddev_nfs_setup.sh" -o "${TMPDIR}/macos_ddev_nfs_setup.sh" || (printf "${RED}Failed downloading "https://raw.githubusercontent.com/${GITHUB_USERNAME}/ddev/master/scripts/macos_ddev_nfs_setup.sh"${RESET}\n" && exit 110)
 
 cd $TMPDIR
 $SHACMD -c "$SHAFILE"
 tar -xzf $TARBALL
```

Comment:
```suggestion
if [ ${VERSION} != "HEAD" ]; then
  tar -xzf ${TARBALL}
else 
  unzip ${TARBALL}
 fi
```

Reply:
HEAD should also work with the `.tar.gz` extension if downloaded that way.

Reply:
If you look at the artifacts for what this is trying to get, you'll see only zip artifacts, see https://nightly.link/drud/ddev/workflows/master-build/master

Why that is I don't know/remember. It would be fine to improve the build process, but that may be out of scope?